src/_Lang_EN.lng

```diff
@@ -261,23 +261,23 @@
        /md5		- calculate md5 hash of files
        /default		- load default settings (they will not be saved)
        /install     	- install HiJackThis to 'Program Files' folder and create shortcuts
-       /autostart   	- set Windows for automatical run the HJT scanning after system boot up (use with /install key)
+       /autostart   	- set Windows to automatically run HJT scanning after system boot up (use with /install key)
        /uninstall	- remove all HiJackThis Registry entries, backups and quit
        /deleteonreboot "c:\file.sys" - delete the specified file after system rebooting using PendingFileRenameOperations mechanism
-       /accepteula 	- accept the agreement. It will not be displayed when program start
+       /accepteula 	- accept the agreement. It will not be displayed when program starts
        /noGUI		- do not show program window during the scan
        /SysTray	- run program minimized in notification area (system tray)
        /LangEN		- force use English language for user interface
        /LangRU		- force use Russian language for user interface
        /LangUA		- force use Ukrainian language for user interface
-       /debug		- tracing mode. Sequence of functions execution will be appended to main log and HiJackThis_debug.log file. You can also rename file to HiJackThis_debug.exe
+       /debug		- tracing mode. Sequence of function execution will be appended to main log and HiJackThis_debug.log file. You can also rename the file to HiJackThis_debug.exe
```

Comment:
It would probably be okay to say "sequence of function **executions**", making "execution" plural.  I'm not an expert, but I believe in this case "execution" is the noun, and "function" is used as an adjective that describes "execution".  So kind of like if you have a lot of red balls, you wouldn't say "reds ball" - you would say "red balls" - in this particular usage, you would say "function executions" not "functions execution".  Hopefully that makes sense?  (It is hard to explain why it "sounds" wrong in native English, sorry!!)

You could also just put "Function executions will be appended to main log..." or maybe even easier "All operations will be appended to main log..." and remove "sequence of function execution" entirely, if that's easier...?

Reply:
Thanks for explanation!